ios/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 5.2.6
+version: 5.2.7
```

Comment:
The build will fail, because you also need to change the framework-references in the .xcconfig file.